Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Cartero/Cartero](https://hosted.weblate.org/projects/cartero/cartero/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/cartero/cartero/horizontal-auto.svg)

franciscoernestoteixeira committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 84.3% (238 of 282 strings)

Translation: Cartero/Cartero
Translate-URL: https://hosted.weblate.org/projects/cartero/cartero/pt_BR/

danirod commented:
Thank you for providing translations to this project, @franciscoernestoteixeira. Consider appending your name or nickname to the marker `translator-credits` string (comma separated), in order to be included in the credits section of the about dialog: https://hosted.weblate.org/translate/cartero/cartero/pt_BR/?checksum=deecda9491124d4f

franciscoernestoteixeira committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 90.4% (255 of 282 strings)

Translation: Cartero/Cartero
Translate-URL: https://hosted.weblate.org/projects/cartero/cartero/pt_BR/

franciscoernestoteixeira committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.9% (265 of 282 strings)

Translation: Cartero/Cartero
Translate-URL: https://hosted.weblate.org/projects/cartero/cartero/pt_BR/
